chipsec/logger.py

```diff
@@ -155,53 +210,31 @@ def __init__(self, profile='debug.conf'):
         self.chipsecLogger.setLevel(logging.DEBUG)
         self.chipsecLogger.addHandler(self.logstream)
         self.chipsecLogger.addHandler(fileH)
-        logging.addLevelName(11, "VERBOSE")
-        logging.addLevelName(12, "HAL")
-        logging.addLevelName(13, "HELPER")
-        logging.setLogRecordFactory(chipsecRecordFactory)  # applies colorization to output
+        self.chipsecLogger.addFilter(chipsecFilter(LOGGER_NAME))
+        logging.addLevelName(level.VERBOSE.value, level.VERBOSE.name)
+        logging.addLevelName(level.HAL.value, level.HAL.name)
+        logging.addLevelName(level.HELPER.value, level.HELPER.name)
+        logFormatter = chipsecLogFormatter('%(additional)s %(message)s')
```

Comment:
```suggestion
        logFormatter = chipsecLogFormatter('%(additional)s%(message)s')
```
Additional space adds a space to the beginning if there's no prefix.